Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/ConceptListEditController.java

```diff
@@ -115,7 +116,8 @@ public String editList(HttpServletRequest req,
 
         // modify the name for all the existing concepts under this concept
         // list
-        List<ConceptEntry> entries = conceptManager.getConceptListEntries(conceptListAddForm.getOldListName());
+        List<ConceptEntry> entries = conceptManager.getConceptListEntries(conceptListAddForm.getOldListName(), 1, -1,
+                "id", IConceptDBManager.DESCENDING);
```

Comment:
I think this is the same issue as for deleting a concept type. Here you are only getting the first page of concepts and change the concept list. For all other concepts in the list, the concept list name doesn't change.